spec/features/reviews/02_create_review_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'add a new review', %{
+  As a user
+  I want to add a review
+  So that I can post my review
+  } do
+  # ACCEPTANCE CRITERIA
+  # * As an authenticated user, I can add a new review
+
+  scenario 'authenticated user adds reviews on park show page' do
+    park = create(:park)
+
+    user = create(:user)
+
+    visit root_path
+    sign_in_as(user)
+
+    visit root_path
+    click_link (park.title)
+
+    fill_in("Title", with: "Awesome park")
+    fill_in("Body", with: "I can go to this park anytime of the year")
+    select "10", from: "Rating"
```

Comment:
Remove this line for now, you're going to have to implement a rating/voting system that will require additional tables